pkg/apis/serving/v1alpha1/route_lifecycle.go

```diff
@@ -67,6 +67,11 @@ func (rs *RouteStatus) MarkServiceNotOwned(name string) {
 		fmt.Sprintf("There is an existing placeholder Service %q that we do not own.", name))
 }
 
+func (rs *RouteStatus) MarkIngressNotConfigured() {
+	routeCondSet.Manage(rs).MarkUnknown(RouteConditionIngressReady,
+		"IngressNotConfigured", "Failed to configure route ingress.")
```

Comment:
Failed seems too strong a wording here since it may just not have reconciled yet.

Reply:
Yeah. Would something like "Route ingress has no status" would be more accurate?

Reply:
`Ingress has not yet been reconciled`?